playground/src/tabs-test.vue

```diff
@@ -7,21 +7,21 @@
 
     <!-- Selected Value -->
     <v-tabs
-      data-testid="Selected Test"
+      data-testid="selected-test"
```

Comment:
[nit] so sorry should've mentioned earlier, but probably also good to have the component type in the test name for clarity, maybe `tabs-selected`, `tabs-default`, `tabs-key-enter`

Reply:
all set